packages/core/lib/feature-flag-delegate.js

```diff
@@ -14,7 +14,16 @@ function add (existingFeatures, name, variant) {
     variant = jsonStringify(variant)
   }
 
-  existingFeatures[name] = variant
+  var found = false
+  for (var i = 0; i < existingFeatures.length; i++) {
```

Comment:
Should we be concerned about the performance of this?  I guess it would be possible to have a secondary data structure (i.e. a map) that allowed quick lookup of whether the feature exists at all, so that you don't need to iterate over all the features every time you add one?

Reply:
https://jsben.ch/cDI8i

interesting, my (admittedly limited) benchmarking shows the array search to be faster than using a Map

Reply:
If you've only got 2 features it'll be basically a wash.  I'm thinking about when you've got 100 or 1000 features.